corehq/apps/hqcase/management/commands/reindex_sql_forms_in_domain.py

```diff
@@ -0,0 +1,22 @@
+from django.core.management import BaseCommand
+
+from corehq.form_processor.backends.sql.dbaccessors import *
+from corehq.pillows.xform import *
+
+
+def reindex_sql_forms_in_domain(domain):
+    for state, _ in XFormInstanceSQL.STATES:
+        doc_ids = FormAccessorSQL.get_form_ids_in_domain_by_state(domain, state)
+        for form in FormAccessorSQL.get_forms(doc_ids):
+            form_json = form.to_json(include_attachments=True)
+            txform = transform_xform_for_elasticsearch(form_json)
+            reindexer = get_sql_form_reindexer()
```

Comment:
Yeah, this seemed to be the most future-proof one though—call the public entry point for reindexing sql forms (`get_sql_form_reindexer`) and then use its `doc_processor`. I don't feel too strongly about it, but I'm inclided to leave as is. Good catch on moving out of the loop